resources/aws/environment.go

```diff
@@ -62,9 +62,16 @@ type Environment struct {
 	randomSubnets pulumi.StringArrayOutput
 }
 
+func (e *Environment) GetInternalRegistry() string {
```

Comment:
The function should be below with other parameters and it should use a parameter set in environment defaults as it's not the same between accounts.

Reply:
For now we always use the QA env registry even for tests run on other account. But it is likely that we will have parameters depending on the account. Updated the PR to pave the way for this use case